src/config.rs

```diff
@@ -2285,6 +2286,7 @@ impl Default for CdcConfig {
             min_ts_interval: ReadableDuration::secs(1),
             old_value_cache_size: 1024 * 1024,
             hibernate_regions_compatible: true,
+            scan_lock_pool_size: 2,
```

Comment:
What is the pool size in before?

Reply:
https://github.com/tikv/tikv/blob/eb57a2aeaeb29a49fe9c8101c8b61d4128d27110/components/cdc/src/endpoint.rs#L284-L291